pkg/result/processors/invalid_issue.go

```diff
@@ -50,3 +52,7 @@ func (p InvalidIssue) shouldPassIssue(issue *result.Issue) (bool, error) {
 
 	return true, nil
 }
+
+func isGoFile(name string) bool {
+	return filepath.Ext(name) == goFileExtension
```

Comment:
Could we possibly consider the necessity of the `goFileExtension` constant?

The following code is readable enough:
```suggestion
	return filepath.Ext(name) == ".go"
```